test/unit/org/apache/cassandra/repair/FuzzTestBase.java

```diff
@@ -776,10 +790,46 @@ public void onFailure(InetAddressAndPort from, RequestFailureReason failureReaso
             }
         }
 
+        private static class CallbackKey
```

Comment:
repair doesn't reuse messages/callbacks so didn't see that this logic was slightly different than `MessagingService`... so need to have both `(message_id, peer)` as the key and not just `message_id`, else the paxos messages clear the callback and hang forever (since it doesn't enqueue work the test actually fails right away).